invenio_rdm_records/services/components/record_deletion.py

```diff
@@ -57,7 +58,7 @@ def restore_record(self, identity, data=None, record=None, **kwargs):
         record.deletion_status = RecordDeletionStatusEnum.PUBLISHED
 
         # Remove the tombstone information
-        record.tombstone = None
+        del record.tombstone
```

Comment:
i've suggested an addition to the tombstone field's documentation, just because the `del` keyword isn't something you see every day anymore

Reply:
personally i would have liked to use some sort of `record.tombstone.remove()` call, but there is some black magic i don't understand. `tombstone` is of type `Tombstone` and not `TombstoneField` so adding `def remove` to `TombstoneField` doesn't work, but adding `def __delete__` to `TombstoneField` and using `del record.tombstone` works.

Reply:
that might be referring to how invenio uses the way python descriptors (which is what systemfields are) work.
you can get the systemfield (i.e. descriptor) itself by accessing the class (e.g. `RDMRecord.tombstone` or `type(record).tombstone`).
if you access the systemfield (i.e. descriptor) via an object of the class, you'll get the tombstone rather than the tombstone field.

you can distinguish between access via object vs. class by virtue of some argument being set; in the case of the tombstone field, the check is here: https://github.com/inveniosoftware/invenio-rdm-records/blob/master/invenio_rdm_records/records/systemfields/tombstone.py#L157